### PR TITLE
[FrameworkBundle] Check for class existence before is_subclass_of

### PR DESCRIPTION
**Pull request by chalasr**
| Q | A |
| --- | --- |
| Branch? | 2.7 |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

Same as #19342

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConsoleCommandPass.php [hunk 1]
```diff
@@ -38,6 +39,10 @@ public function process(ContainerBuilder $container)
 
             $class = $container->getParameterBag()->resolveValue($definition->getClass());
             if (!is_subclass_of($class, 'Symfony\\Component\\Console\\Command\\Command')) {
+                if (!class_exists($class, false)) {
+                    throw new InvalidArgumentException(sprintf('Class "%s" used for service "%s" cannot be found.', $class, $id));
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConsoleCommandPass.php]: we should use `\InvalidArgumentException` for consistency in this PR (and rely on your other PR to use "DI" exceptions)

Comment (chalasr) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConsoleCommandPass.php]: I thought the inverse would be easier, change done

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by nicolas-grekas** [comment]
Thank you @chalasr.

**Pull request merged**